package.json

```diff
@@ -29,9 +25,33 @@
   "publishConfig": {
     "registry": "https://registry.npmjs.org"
   },
+  "packageManager": "pnpm@9.7.0",
+  "pnpm": {
+    "packageExtensions": {
+      "ember-cli-fastboot-testing": {
+        "peerDependencies": {
+          "@ember/test-helpers": "^2.0.0 || ^3.2.0"
+        },
+        "peerDependenciesMeta": {
+          "@ember/test-helpers": {
+            "optional": true
+          }
+        }
+      }
+    },
+    "peerDependencyRules": {
+      "allowAny": [
+        "ember-runtime-config",
+        "typescript"
+      ],
+      "ignoreMissing": [
```

Comment:
@vstefanovic97 why do we need this? where `webpack` is missing as dep?

Reply:
![image](https://github.com/user-attachments/assets/7616f866-fc5b-44d3-a015-ae3c3cce46ee)
I really doubt we should be actually adding webpack as a devDep there

Reply:
@vstefanovic97 thank you for explanation!